WeatherApp/WeatherApp.Webpage/Services/WeatherForecastService.cs

```diff
@@ -41,25 +43,23 @@ public IList<WeatherForecast> GetForecasts(string city)
             }
 
             var json = JObject.Parse(jsonString).GetValue("list");

```

Comment:
Instead of JObject Parse use standard JSON serialization mechanism. The `IHttpClientFactory` article has an example for that as well.